.gitignore

```diff
@@ -1,3 +1,3 @@
-formicactl
+inagoctl
```

Comment:
I'd rather not, it means we'll have it hanging around. I'm sure we'll notice in the PR if someone is committing binaries.